images/cross-cloud/sles/chost/15-sp1/content.yaml

```diff
@@ -44,9 +47,14 @@ image:
       _include:
         - csp/openstack/settings/chost
         - products/chost
+    - attributes:
+        profiles: [SAP Converged Cloud]
```

Comment:
The profile name cannot have spaces, I suggest using `[SAP-Converged-Cloud]`. The profile definition is used to generate a `_multibuild` file for the Open Build Service. That string in turn is used when the build service executes kiw and as such would end up looking like `--profile SAP Converged Cloud`, which in turn will trip up the command line parser for kiwi due to the spaces.

Reply:
Or maybe even just `SAPCC`.

Reply:
I have to go here with the SAP naming convention. So `SAP-CCloud` it is.